core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -18,22 +18,30 @@
 package kafka.admin
 
 import java.text.{ParseException, SimpleDateFormat}
+import java.util
 import java.util.{Date, Properties}
+
 import javax.xml.datatype.DatatypeFactory
 import joptsimple.{OptionParser, OptionSpec}
+import kafka.common.{OffsetMetadataAndError, TopicAndPartition}
 
 
 import kafka.utils._
-import kafka.utils.Implicits._
+import org.I0Itec.zkclient.exception.ZkNoNodeException
```

Comment:
Did we need this import? There are a few more unneeded imports to clean up as well.